crates/jit/src/link.rs

```diff
@@ -400,17 +391,31 @@ fn relocate(
 
 /// A declaration for the stack probe function in Rust's standard library, for
```

Comment:
Looks like this needs to be changed to a regular comment or moved elsewhere based on the build error.